src/styles/block/header.scss

```
@@ -0,0 +1,56 @@
+@import "./src/styles/mixins";
+@import "./src/styles/variables";
+
+.header {
+  height: 60px;
+  width: 100%;
+  padding: 0 50px;
+  margin-bottom: 50px;
+  @include display-justify-content;
+  box-shadow: 0 2px 4px $box-shadow-c;
+}
+
+.logo {
+  position: relative;
+  top: 10px;
+}
+
+.nav__list {
+  @include display-justify-content;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  list-style-type: none;
+}
+
+.nav__link {
+  display: block;
+  margin-right: 20px;
+  line-height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: black;
+}
+
+.nav__video {
+  margin-right: 0;
+}
```

```suggestion
.nav {
  &__item {
     &:last-child {
         margin-right: 0;
         }
    }
 }
```
можна зробити так і не додавати зайві класи до крайнього елементу